.github/workflows/lint.yaml

```diff
@@ -87,3 +87,11 @@ jobs:
             - name: Check if any dependencies are compromised
               if: always() && steps.install.outcome == 'success'
               run: /home/runner/.symfony/bin/symfony check:security
+
+            - name: Install PHPUnit
```

Comment:
If phpunit is installed because one of the linting steps depends on it, it should be part of the installation section (which is checked with `steps.install.outcome == 'success'` in linting steps to ensure that the install worked before running linting steps).

Note that `id: install` must be set on the last step belonging to the installation.

Reply:
@stof Thanks, I didn't notice there was kind of a 'installation section'. I moved the phpunit installation step there in 05fce162699163f5ccb64fbab2ae3cf4273d883f.

Shouldn't the 'Download Symfony CLI' step also be part of that 'installation section'?